src/Data/Food/Builder/Recipe.elm

```diff
@@ -531,35 +530,6 @@ deletePackaging code query =
     }
 
 
-encodeIngredient : BuilderQuery.IngredientQuery -> Encode.Value
-encodeIngredient i =
-    Encode.object
-        [ ( "id", Ingredient.encodeId i.id )
-        , ( "mass", Encode.float (Mass.inKilograms i.mass) )
-        , ( "variant", variantToString i.variant |> Encode.string )
-        , ( "country", i.country |> Maybe.map Country.encodeCode |> Maybe.withDefault Encode.null )
-        ]
-
-
-encodeProcess : BuilderQuery.ProcessQuery -> Encode.Value
-encodeProcess p =
-    Encode.object
-        [ ( "code", p.code |> Process.codeToString |> Encode.string )
-        , ( "mass", Encode.float (Mass.inKilograms p.mass) )
-        ]
-
-
-encode : Query -> Encode.Value
-encode q =
-    Encode.object
-        [ ( "ingredients", Encode.list encodeIngredient q.ingredients )
-        , ( "transform", q.transform |> Maybe.map encodeProcess |> Maybe.withDefault Encode.null )
-        , ( "packaging", Encode.list encodeProcess q.packaging )
-        , ( "preparation", Encode.list Preparation.encodeId q.preparation )
-        , ( "distribution", q.distribution |> Maybe.map Retail.encode |> Maybe.withDefault Encode.null )
-        ]
-
-
```

Comment:
Removal of unused code? Nice one, I wonder if it would be possible to catch those early on? Or is it code that was used up til now?